src/Appwrite/Platform/Tasks/Restore.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Exception;
+use Utopia\App;
+use Utopia\DSN\DSN;
+use Utopia\Platform\Action;
+use Utopia\CLI\Console;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\DOSpaces;
+use Utopia\Storage\Device\Local;
+use Utopia\Validator\Text;
+use Utopia\Validator\WhiteList;
+
+class Restore extends Action
+{
+    protected ?DSN $dsn = null;
+    protected string $database;
+    public const PROCESSORS = 4;
+
+    public static function getName(): string
+    {
+        return 'restore';
+    }
+
+    public function __construct()
+    {
+        $this->checkEnvVariables();
+
+        $this
+            ->desc('Restore a DB')
+            ->param('id', '', new Text(20), 'Folder Identifier')
+            ->param('cloud', null, new WhiteList(['true', 'false'], true), 'Take file from cloud?')
```

Comment:
I think we can use the `Boolean` validator with `loose = true` to cast from strings here instead of `WhiteList`

Reply:
There is a bug using the value  false, That's why I was using a string

Reply:
Yes loose is ok 👍  , with adding a small check
$cloud = $cloud === 'true' || $cloud === '1';

Reply:
I think those should already be handled in the validator: https://github.com/utopia-php/framework/blob/master/src/Validator/Boolean.php#L76-L82

Reply:
The param is working find, I mean this:
https://github.com/appwrite/appwrite/pull/5878/files#diff-466e170fc0ce353e735b9c582d7be317be85c8f934f3a69ef00ffe708daa89a4R69